trunk/SUAVE/Analyses/Aerodynamics/AVL.py

```diff
@@ -116,8 +121,10 @@ def initialize(self):
         self.geometry
         """          
         self.process.compute.lift.inviscid.geometry = self.geometry
+        sv = self.settings.spanwise_vortices
```

Comment:
Unpack these up top

Reply:
Moved to the top and inserted #unpack comment